docs/index.rst

```diff
@@ -0,0 +1,376 @@
+.. troposphere documentation master file, created by
+   sphinx-quickstart on Fri Feb  1 14:27:25 2019.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+===========
```

Comment:
This appears to be a copy of README.rst from the top-level. Rather than keeping this in sync, perhaps it could be changed to just do:
```
.. include:: ../README.rst
```
This also begs the question as to whether the TOC should come first.

Reply:
Concur with the include it is changed/done.  The TOC is now first, this to me is a matter of taste as the TOC is also in the sidebar.

Reply:
Yeah, I wasn't sure which one looked better. I'll let you decide on where the TOC goes. :-) Feel free to push the "include" change and I'll merge.

Reply:
After looking at each several times, I went with TOC at the bottom by index.  It changed be changed easily, especially since the README is now included, thanks for that tidbit.